### ISSUE TITLE
case-sensitive rule issue with webpack --watch

### ISSUE THREAD

**Issue by quentinneyraud** [zertosh/eslint-plugin-dependencies#14]
Hi @zertosh ,

I'm using `eslint-plugin-dependencies` with eslint-loader in a webpack configuration and i found a bug when i execute `webpack --watch`.

**Step to reproduce :**

- Create a webpack configuration with eslint-loader  

```json
// my .eslintrc file

{
    "extends": [
        "standard"
    ],
    "plugins": [
        "dependencies"
    ],
    "rules": {
        "indent": [
            "warn",
            4
        ],
        "dependencies/case-sensitive": 1,
        "dependencies/no-cycles": 1,
        "dependencies/no-unresolved": 0,
        "dependencies/require-json-ext": 1
    },
    "globals" : {
        "DEVELOPMENT": false,
        "PRODUCTION": false,
        "ENV": false
    }
}
```
- Create your webpack entry file `js/main.js`
- Create a file `js/utils/a.js`
- Import `js/utils/a.js` in `js/main.js`
```js
// js/main.js

import a from './utils/a.js'
```

- Start webpack with `--watch`argument
- Create a file `js/utils/b.js`
- Import `js/utils/b.js` in `js/main.js`
```js
// js/main.js

import a from './utils/a.js'
import b from './utils/b.js'
```

This should report `dependencies/case-sensitive` error.
`_readdirCache` is filled on first webpack run with something like:
```javascript
_readdirCache['utils'] = [
   'a.js'
]
```
and it's never updated.  

This lines work :
```js
// case-sensitive.js

// var _readdirCache = Object.create(null);
function readdirSync(dirname) {
  // if (!(dirname in _readdirCache)) {
  //   try {
  //     _readdirCache[dirname] = fs.readdirSync(dirname);
  //   } catch (err) {
  //     _readdirCache[dirname] = null;
  //   }
  // }
  // return _readdirCache[dirname];
  fs.readdirSync(dirname)
}
```

**Comment by zertosh** [comment]
published the fixed in eslint-plugin-dependencies@2.4.0
